pkg/object/object.go

```diff
@@ -236,9 +295,85 @@ func (v *Volume) validate() error {
 	return nil
 }
 
+func (sd *SecretData) validate() error {
```

Comment:
> Where do you validate if the field Base64 is a valid base64 encoded value?

We cannot validate that - https://stackoverflow.com/questions/8571501/how-to-check-whether-the-string-is-base64-encoded-or-not#comment23919648_8571649

> Similarly for field File, it should be a valid path, either absolute or relative depending on what is supported!

We need the path of the OpenCompose file, which we do not have in object.go. This case gets covered in encoding.go, when the absolute secret path gets generated.